project/build.scala

```diff
@@ -125,18 +135,17 @@ object BananaRdfBuild extends Build {
   lazy val rdfTestSuite_jvm = rdfTestSuiteM
     .project(Jvm, rdfTestSuite_common_jvm)
     .settings(Seq(resourceDirectory in Test := baseDirectory.value / "src/main/resources"):_*)
-    .dependsOn(rdf_jvm)
+    .dependsOn(rdf_jvm,ntriples_jvm)
 
   lazy val rdfTestSuite_js = rdfTestSuiteM
     .project(Js, rdfTestSuite_common_js)
     .settings(Seq(resourceDirectory in Test := baseDirectory.value / "src/main/resources"): _*)
-    .dependsOn(rdf_js)
+    .dependsOn(rdf_js,ntriples_js)
```

Comment:
In fact, it is! When I remove this dependency the test project does not compile for some strange reason. If you can remove it without breaking anything, then.... git is distributed: PR into my PR is welcome=) 
Anyway, removal of ntriples_js dependency is not in scope of "fixing the build for IDEA", it can be a part of another PR